### DISCUSSION TITLE
How to change the rate of cursor blinking in NetBeans?

### DISCUSSION THREAD

**Discussion by Ahmed-142** [apache/netbeans#6818] [Q&A] [answered]
Hello everyone,

How to change the rate of cursor blinking in NetBeans to make the blinking rate is more slower the the curent blinking rate?

I use Windows 10 and I use Apache NetBeans IDE 20

**Comment by mbien** [comment, marked as answer]
I don't think there is a setting for this in the UI but you can change it in the config file.

open:
`[netbeans-config]/[version]/config/Editors/Preferences/org-netbeans-modules-editor-settings-CustomPreferences.xml`

and add this entry if it isn't there yet:
```xml
    <entry javaType="java.lang.Integer" name="caret-blink-rate" xml:space="preserve"><value>1000</value></entry>
```
the value represents the blinking delay in ms.

The `Help->About` window would dispaly the config folder location.

**Comment by Ahmed-142** [comment]
My problem is solved since some days, so I'm sorry for replying late, actually what I did is by depending on the following website:
[https://stackoverflow.com/questions/6324058/how-to-change-caret-cursor-blinking-rate-in-netbeans](url)

and what I did is same or similar to what you said.

Thank you so much for your help.

**Reply by Ahmed-142** [reply]
I don't know why in the last replay I sent before this replay when I click on the link, the link does not work, but anyway I copied the link and I pasted it in the address bar and it works.
